brightener.cpp

```diff
@@ -1,6 +1,6 @@
 #include "brightener.h"
```

Comment:
Try pasting this entire file to chatgpt and ask it to reduce the duplication. If its code has errors, continue the conversation by telling it to correct the error 
Let me know if you could rework the code with its suggestions